ui/desktop/electron-app/config/desktop.js

```diff
@@ -11,11 +25,19 @@ const createConfig = () => {
     releaseVersion: process.env.RELEASE_VERSION,
     releaseCommit: process.env.RELEASE_COMMIT,
     executableName: 'Boundary',
+    releaseVersionRaw: process.env.RELEASE_VERSION,
```

Comment:
@ZedLi As [per your comment](https://github.com/hashicorp/boundary-desktop-releases/pull/128#discussion_r1953494199) I think this can be a good approach.

I created the `releaseVersionRaw` which as its name shows, its the same as `releaseVersion` but raw, without extracting the semVer.

`releaseVersionRaw` its ONLY use by electron-forge, within the `postMake` hook to copy artifacts.

Let me know your thoughts.